test/SharedEventManagerTest.php

```diff
@@ -27,7 +27,7 @@ public function setUp()
 
     public function getListeners(SharedEventManager $manager, array $identifiers, $event, $priority = 1)
     {
-        $priority = (int) $priority . '.0';
+        $priority = (int) $priority;
```

Comment:
Which may be a rationale for re-instating the getListeners() method. That said, it could be marked private, and libraries could use reflection in order to query it.

This is me brainstorming via github comments. Sue me.